source/_components/ihc.markdown

```diff
@@ -0,0 +1,90 @@
+---
+layout: page
+title: "IHC"
+description: "Instructions on how to integrate the IHC components with Home Assistant"
+date: 2017-11-11 22:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ihc.png
+ha_category: Hub
+ha_release: "0.62"
+ha_iot_class: "Local Push"
+---
+
+IHC Controller integration for Home Assistant allows you to connect the LK IHC controller to Home Assistant. 
+(The controller is sold under other names in different countries - "ELKO Living system" in Sweden and Norway)
+
+An `ihc` section must be present in the `configuration.yaml` file and contain the following options:
+
+```yaml
+# Example configuration.yaml entry
+ihc:
+   host: http://192.168.1.3
+   username: admin
+   password: mysecret
+   auto_setup: True
+   info: True
+```
+
+{% configuration %}
+auto_setup:
+  description: True to have IHC products auto setup.
+  required: false
+  type: bool
+host:
+  description: The URL of the IHC Controller.
+  required: true
+  type: string
+info:
+  description: If True additional IHC info will be shown on each component.
+  required: false
+  type: bool
+password:
+  description: The password for the IHC Controller.
+  required: true
+  type: string
+username:
+  description: The username for the IHC Controller.
+  required: true
+  type: string
+{% endconfiguration %}
+
+The info option will show the IHC "name", "note" and "position" attributes. 
+This will make it easier to identify the IHC products within Home Assistant
+
+There is currently support for the following device types within Home Assistant:
+
+- [Binary Sensor](../binary_sensor.ihc/) 
+- [Sensor](../sensor.ihc/) 
+- [Light](../light.ihc/) 
+- [Switch](../switch.ihc/) 
+
+### Auto setup of IHC products
+
+If auto setup is enabled, the `ihc` component will automatically find IHC products and insert these as devices in Home Assistant.
+To disable this set auto_setup to False. (Auto setup is on by default)
+See the individual device types for a list of IHC products to be recognized automatically.
+
+Components will get a default name that is a combination of the IHC group and IHC resource id.
+If you want to change the display names use the [Customizing entities](/docs/configuration/customizing-devices/)
+
+### {% linkable_title Manualy setup %}
```

Comment:
"Manual setup"
Make sure links to anchor are adjusted.

Reply:
Done